src/mobility/components/filter/MobilityFilters.tsx

```diff
@@ -0,0 +1,104 @@
+import {ScooterFilter} from '@atb/mobility/components/filter/ScooterFilter';
+import {BikeFilter} from '@atb/mobility/components/filter/BikeFilter';
+import {CarFilter} from '@atb/mobility/components/filter/CarFilter';
+import {Section} from '@atb/components/sections';
+import React, {useEffect, useState} from 'react';
+import {useIsCityBikesEnabled, useIsVehiclesEnabled} from '@atb/mobility';
+import {useIsCarSharingEnabled} from '@atb/mobility/use-car-sharing-enabled';
+import {MapFilterType, OperatorFilterType} from '@atb/components/map';
+import {StyleSheet} from '@atb/theme';
+import {useSafeAreaInsets} from 'react-native-safe-area-context';
+
+type Props = {
+  initialFilter: MapFilterType;
+  onFilterChanged: (filter: MapFilterType) => void;
+};
+
+export const MobilityFilters = ({initialFilter, onFilterChanged}: Props) => {
```

Comment:
Hmm... I see your point. Good spot 👍  React noob mistake.

Not sure if I see the best solution, though. Ideally, I don't want all details about mobility filter stuff to bleed all the way out to `MapFilters`.

Reply:
Any suggestions?

Reply:
I've made a change suggestion now (see my latest commit, https://github.com/AtB-AS/mittatb-app/pull/3596/commits/ef0b6069fc882fee3041abe55746af3e0c2c329b).

I don't think it's too bad. Updating the `MapFilterType` should probably be owned by `MapFilterSheet` anyways.

Reply:
> I've made a change suggestion now

I think it looks good. A little more verbose, but it is more correctly implemented. 

Another strategy for cases like this is to split it into a React Component and hook. That would mean `MobilityFilters` only has the responsibility to show the toggles, and sync the toggle state to the parent. And then it would be a `useMobilityFilters` (or something like it) which takes in the toggle states and has the responsibility for applying them correctly and returning the state with the correct vehicles. Not necessarily better to do it like this here now, but just mentioning it.

> Any suggestions?

Just reach out on Slack if something should be discussed right away. I try to take a look at PRs at morning and after lunch, so will not always respond to comments immediately.

Reply:
Ok, I'll leave it like this for now. 

I'll wait until after the release branch has been made to merge this. It requires both BFF and config changes, and it is strictly a part of next sprint anyways.